test/common/testlib.py

```diff
@@ -672,19 +704,10 @@ def open_session_menu(self):
             self.click("#toggle-menu")
 
     def open_superuser_dialog(self):
-        if self.cdp.mobile:
-            self.open_session_menu()
-            self.click("#super-user-indicator-mobile button")
```

Comment:
This means that with this PR we completely lost testing the superuser indicator in mobile mode? I don't see any replacement for it.

Reply:
True.  I add one.

Reply:
Done.  OPening the dialog in "mobile" brings back the title tooltip. Oh well.